### DISCUSSION TITLE
Move items without activators

### DISCUSSION THREAD

**Discussion by naman-ali** [clauderic/dnd-kit#422] [Q&A] [answered]
I am struggling to move draggables without using activators. 

Expected behavior:
Using keyboard sensor, a user does not press Space to be able to moving items using arrow keys. I.e., a user first use Tab to select/activate different items, and by pressing arrow keys the item move to another position. It can be seen as activate items by default when Tab is pressed (the item is in focus mode).

I added Tab in defaultKeyboardCodes for start property, but still the user should press tab 2 times time to make the draggable activated.

**Comment by clauderic** [comment, marked as answer]
You can create a new sensor that extends the `KeyboardSensor`. Override the activator to be `onFocus` rather than `onKeyDown`:

```ts
import {KeyboardSensor} from '@dnd-kit/core';

class FocusKeyboardSensor extends KeyboardSensor {
  static activators = [
    {
      eventName: 'onFocus',
      handler(event) {
        return true;
      }
    }
  ]
}
```
